v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/README_Load_Testing.md

```diff
@@ -0,0 +1,142 @@
+   properties:
+    1. Create new subnet with `Private Google Access` as On
+    2. Set `Private Google Access` as **Global**
+
+```shell
+export PROJECT_NAME=<my-project>
+export NETWORK_NAME=<my-network>
+export SUBNET_NAME=<my-subnet>
+
+gcloud compute networks create ${NETWORK_NAME} --project=${PROJECT_NAME} \
+ --subnet-mode=custom --mtu=1460 --bgp-routing-mode=global 
+ 
+gcloud compute networks subnets create ${SUBNET_NAME} --project=${PROJECT_NAME} \
+--range=10.0.0.0/24 --stack-type=IPV4_ONLY --network=${NETWORK_NAME} --region=us-central1 \
+--enable-private-ip-google-access
+```
+
+2. Modify SQL Instance and configure the VPC network created in the previous step to the Cloud SQL Instance by
+   following this [guide](https://cloud.google.com/sql/docs/mysql/configure-private-ip?_gl=1*ocwud2*_ga*MTI0MzA5MzM4My4xNzE4OTQ3NzU1*_ga_WH2QY8WWF5*MTcyMTIzOTUxNi42LjEuMTcyMTI0MzgzMC4zLjAuMA..#existing-private-instance)
+```shell
+export CLOUD_SQL_INSTANCE_ID=<my-instance>
+export PROJECT_NAME=<my-project>
+export NETWORK_PROJECT_ID=<my-network-project>
+export NETWORK_NAME=<my-network>
+
+gcloud beta sql instances patch ${CLOUD_SQL_INSTANCE_ID} \
+--project=${PROJECT_ID} \
+--network=projects/${NETWORK_PROJECT_ID}/global/networks/${NETWORK_NAME} \
+--no-assign-ip \
+--enable-google-private-path
+```
+
+3. Create tables in the instance by either connecting to it through the
+   [shell](https://cloud.google.com/sql/docs/mysql/connect-instance-cloud-shell) or through
+   [Cloud Sql Studio](https://cloud.google.com/sql/docs/mysql/manage-data-using-studio).
+
+   [**Sample MySQL Schema**](https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/datastream-to-spanner/src/test/resources/DataStreamToSpannerLoadTesting/mysql-schema.sql)
+
+4. Create a schema file that contains a JSON template for the generated data. For more information, see the
+   [json-data-generator documentation](https://github.com/vincentrussell/json-data-generator/blob/master/README.md). Sample
+   can be found [here](https://github.com/vincentrussell/json-data-generator/blob/master/README.md). Upload the schema file
+   to a Cloud Storage bucket.
+
+   [**Sample Schema File**](https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/datastream-to-spanner/src/test/resources/DataStreamToSpannerLoadTesting/datagenerator-schema.json)
+
+5. Run the template by following this [guide](https://cloud.google.com/dataflow/docs/guides/templates/provided/streaming-data-generator)
+```shell
+export PROJECT=<my-project>
+export BUCKET_NAME=<bucket-name>
+export REGION=<region>
+export TEMPLATE_SPEC_GCSPATH="gs://dataflow-templates-${REGION}/latest/flex/Streaming_Data_Generator"
+export QPS=<qps>
+export MESSAGE_LIMIT=<row-count>
+export SCHEMA_LOCATION=<schemaLocation>
+export DRIVER_CLASS_NAME=<driverClassName>
+export CONNECTION_URL=<connectionUrl>
+export USERNAME=<username>
+export PASSWORD=<password>
+export STATEMENT=<statement>
```

Comment:
Its unclear what some of these params do: what is statement? Default for message limit if i want unlimited. If we can add a link to this, then thatd make things easier.